posts/2022-02-21-rust-analyzer-joins-rust-org.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "rust-analyzer joins the Rust organization!"
+author: The rust-analyzer Team on behalf of the entire Rust Team
+---
+
+We have an exciting announcement to make!
+The [rust-analyzer](https://rust-analyzer.github.io) project, a new implementation of the Language Server Protocol (LSP) for Rust, is now officially a part of the wider Rust organization! :tada:
+
+We want to start by thanking everyone who has gotten us this far from contributors, to sponsors, to all the users of rust-analyzer in the Rust community. We could not have done this without you.
+
+The immediate impact of this organizational change is limited -- nothing changes for rust-analyzer users or contributors.
+However, this change unblocks technical work to make rust-analyzer the officially recommended language server for Rust in the near future.
+
+If you were hesitant to try rust-analyzer before, today is a good opportunity to do so.
+Not only is it a very capable language server for Rust, but according to [VS Code statistics](https://marketplace.visualstudio.com/search?target=VSCode&category=Programming%20Languages&sortBy=Rating), it is one of the best rated LSP implementations *across* programming languages.
+We highly recommend giving rust-analyzer a spin today, even if it will take some more time for us to complete the due process and switch from the existing officially recommended LSP implementation (RLS) properly.
+
+rust-analyzer enjoys first-class support in many editors:
+
+* For VS Code, install the [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=matklad.rust-analyzer) extension from the marketplace.
+* For Neovim, follow the setup instructions [from this post](https://sharksforarms.dev/posts/neovim-rust/).
+  For Vim, see [coc-rust-analyzer](https://github.com/fannheyward/coc-rust-analyzer).
```

Comment:
```suggestion
* For Vim, see [coc-rust-analyzer](https://github.com/fannheyward/coc-rust-analyzer).
```

(Might be good to check the rendering through our markdown engine -- probably this should be on its own line, at least, if not necessarily with a separate bullet).

Reply:
This intedns to be a single Vim/Neovim bullet.

Reply:
![image](https://user-images.githubusercontent.com/1711539/154303226-3a7b0a1e-33fb-41fd-851c-3b33630386fe.png)

Yup, this is how I want this to look!